cves/CVE-2018-6912.yml

```diff
@@ -344,7 +365,20 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    Technically this vulnerability occured due to a lack of distrust between the
+    developer and the input data. While the codec may have been implemented due to
```

Comment:
change to "of the input data" to be more concise.

Reply:
Thanks!

Fixed in 741b8b5